demo/app.py

```diff
@@ -51,6 +51,26 @@
             CONTRACT_TYPES[0],
             id="ctr-type",
         ),
+        html.Br(),
+        dcc.RadioItems(
+            options=[
+                {'label': 'Call', 'value': 'Call'},
```

Comment:
if you use True, False as value here, then you don't need to do the conversion later.